sdk/servicebus/Azure.Messaging.ServiceBus/tests/ServiceBusTestBase.cs

```diff
@@ -9,15 +9,6 @@ namespace Azure.Messaging.ServiceBus.Tests
 {
     public class ServiceBusTestBase
     {
-        protected static string ConnString = Environment.GetEnvironmentVariable("SERVICE_BUS_CONN_STRING", EnvironmentVariableTarget.Machine);
-        protected static string TenantId = Environment.GetEnvironmentVariable("TENANT_ID", EnvironmentVariableTarget.Machine);
-        protected static string ClientId = Environment.GetEnvironmentVariable("CLIENT_ID", EnvironmentVariableTarget.Machine);
-        protected static string ClientSecret = Environment.GetEnvironmentVariable("CLIENT_SECRET", EnvironmentVariableTarget.Machine);
-        protected const string QueueName = "josh";
-        protected const string SessionQueueName = "joshsession";
-        protected const string TopicName = "joshtopic";
```

Comment:
Pretty sure "joshtopic" sounds like a cool place in the 90s to buy trendy clothes....
![image](https://user-images.githubusercontent.com/913445/74351819-a310d580-4d85-11ea-9a2b-0045bacead98.png)

Reply:
Hah, I thought that sounded familiar.